### ISSUE TITLE
Error when building

### ISSUE THREAD

**Issue by Staz0r** [chrieke/mkdocs-exclude-search#25]
Hey, I just installed the plugins, but when I tried to build it, it returns this error
```python Traceback (most recent call last):
  File "..\Python\Python 3.9\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "..\Python\Python 3.9\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "..\Python\Python 3.9\Scripts\mkdocs.exe\__main__.py", line 7, in <module>
  File "..\Python\Python 3.9\lib\site-packages\click\core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "..\Python\Python 3.9\lib\site-packages\click\core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "..\Python\Python 3.9\lib\site-packages\click\core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "..\Python\Python 3.9\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "..\Python\Python 3.9\lib\site-packages\click\core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "..\Python\Python 3.9\lib\site-packages\mkdocs\__main__.py", line 187, in build_command
    build.build(config.load_config(**kwargs), dirty=not clean)
  File "..\Python\Python 3.9\lib\site-packages\mkdocs\commands\build.py", line 317, in build
    config['plugins'].run_event('post_build', config=config)
  File "..\Python\Python 3.9\lib\site-packages\mkdocs\plugins.py", line 104, in run_event
    result = method(**kwargs)
  File "..\Python\Python 3.9\lib\site-packages\mkdocs_exclude_search\plugin.py", line 256, in on_post_build
    navigation_items = [
  File "..\Python\Python 3.9\lib\site-packages\mkdocs_exclude_search\plugin.py", line 257, in <listcomp>
    list(nav_chapter.values())[0].replace(".md", "/")
AttributeError: 'list' object has no attribute 'replace'
```
This is the config that I used for the build
```yml
  - exclude-search:
      exclude:
        - contribute.md
      exclude_tags: true
      exclude-unreferenced: true
```

**Comment by chrieke** [comment]
Thank you @Staz0r I must have introduced this issue yesterday with version 0.6.0. It's not easy to debug since with my example config it works. I'm suspecting it has to do with multiple subchapters in the mkdocs.yml nav section, checking that now.

**Comment by chrieke** [comment]
@Staz0r Fixed with https://github.com/chrieke/mkdocs-exclude-search/pull/26 and released in 0.6.1. Please test, could replicate the issue and this should resolve it. Thanks for reporting!
